warehouse/macros/define_gtfs_guidelines.sql

```diff
@@ -11,6 +11,14 @@
 "Includes complete wheelchair accessibility data in both stops.txt and trips.txt"
 {% endmacro %}
 
+{% macro shapes_file_present() %}
+"Shapes.txt file is present"
+{% endmacro %}
+
+{% macro shapes_valid() %}
+"No shapes-related errors appear in the MobilityData GTFS Validator."
```

Comment:
Looks like we haven't been using periods in general, may drop the one here for consistency